src/levels.rs

```diff
@@ -1,6 +1,10 @@
 use std::ffi::c_uint;
 use std::ops::Neg;
 
+use strum::EnumCount;
+
+use crate::src::enum_map::DefaultValue;
```

Comment:
Could you remove the extra lines between `use` statements so that imports are grouped and can be automatically sorted?

```suggestion
use strum::EnumCount;
use crate::src::enum_map::DefaultValue;
```